src/open_inwoner/haalcentraal/utils.py

```diff
@@ -83,24 +91,49 @@ def update_brp_data_in_db(user, brp_version, initial=True):
 
```

Comment:
Oddly the line I want to comment doesn't show up in this view (only if you follow "show file").

Instead of
```
 if data.get("personen"):
     data = data.get("personen", [])[0]
 else:
     data = []
```
perhaps the following:
```
if data := data.get("personen", []):
    data = data[0]
```
then you can ditch the `else` clause.